draft-jasdips-regext-rdap-rpki.md

```diff
@@ -74,72 +75,77 @@ Furthermore, correlating registered RPKI data with registered IP networks and au
+utility of such an approach.
 
-This specification next defines RDAP object classes, as well as lookup and search path segments, for the ROA, ASPA, and
-BGPSec Router Certificate registration data.
+For these purposes, this specification defines RDAP object classes, as well as lookup and search path segments, for the
+ROA, ASPA, and BGPSec router certificate registration data.
 
 ## Requirements Language
 
-The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
-"SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and
-"OPTIONAL" in this document are to be interpreted as described in BCP
-14 [@!RFC2119] [@!RFC8174] when, and only when, they appear in all
-capitals, as shown here.
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",
+"NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [@!BCP14] when, and only
+when, they appear in all capitals, as shown here.
 
-Indentation and whitespace in examples are provided only to illustrate
-element relationships, and are not a REQUIRED feature of this
-protocol.
+Indentation and whitespace in examples are provided only to illustrate element relationships, and are not a REQUIRED
+feature of this protocol.
 
-"..." in examples is used as shorthand for elements defined outside of
-this document.
+"..." in examples is used as shorthand for elements defined outside of this document.
 
 # Common Data Members {#common_data_members}
 
-An RDAP object class for RPKI in (#roa_object_class), (#aspa_object_class), and (#bgpsec_router_cert_object_class) can
+The RDAP object classes for RPKI ((#roa_object_class), (#aspa_object_class), (#bgpsec_router_cert_object_class)) can
 contain one or more of the following common members:
 
-* handle -- a string representing the registry unique identifier of the RPKI object registration
-* name -- a string representing an identifier assigned to the RPKI object registration by the registration holder
-* notValidBefore -- a string that contains the time and date in Zulu (Z) format with UTC offset of 00:00 ([@!RFC3339]), 
-  representing the not-valid-before date of the end-entity certificate for the RPKI object ([@!RFC6487, section 4])
-* notValidAfter -- a string that contains the time and date in Zulu (Z) format with UTC offset of 00:00 ([@!RFC3339]), 
-  representing the not-valid-after date of the end-entity certificate for the RPKI object ([@!RFC6487, section 4])
-* autoRenewed -- a boolean indicating if the registered RPKI object is auto-renewed if true
-* publicationUri -- a URI string pointing to the location of the RPKI object within the RPKI repository; 
+* "handle" -- a string representing the registry unique identifier of an RPKI object registration
+* "name" -- a string representing the identifier assigned to an RPKI object registration by the registration holder
```

Comment:
`handle` and `name` will not be available in all systems. The RIPE NCC CA does not expose this kind of identifier to the user at the moment.

Reply:
Good point. For any RDAP object class, all members beside "objectClassName" are considered optional. We could label each member as REQUIRED/OPTIONAL but have shied away from that for now.

Reply:
Logged this gap as issue #18.